q2_longitudinal/tests/test_longitudinal.py

```diff
@@ -361,6 +361,12 @@ def test_volatility(self):
             output_dir=self.temp_dir.name, metadata=self.md_ecam_fp,
             state_column='month', individual_id_column='studyid')
 
+    def test_volatility_no_individual_id_column(self):
```

Comment:
This PR gets us in the business of conditionally rendering the vega spec - how about we do a check in this test to make sure that the portions of the spec we expect to be missing are in fact gone now. For example, you could ensure that the substring `spaghettiLine` isn't present (I think - you should double check me on that).